src/Docker.DotNet/Models/ServiceFilter.cs

```diff
@@ -9,30 +9,30 @@ public class ServicesListParameters
         [QueryStringParameter("filters", false, typeof(MapQueryStringConverter))]
         public ServiceFilter Filters { get; set; }
     }
-
-    public class ServiceFilter : Dictionary<string, string>
+    
+    public class ServiceFilter : Dictionary<string, string[]>

```

Comment:
This is a breaking change. Did docker break it?

Reply:
Haven't found the reference, but it fails when left unmodified
```
 Docker.DotNet.DockerApiException : Docker API responded with status code=BadRequest, response={"message":"json: cannot unmarshal string into Go value of type map[string]bool"}
```

Reply:
Yep its now a `map[string][]string` your right